src/pages/workspace/accounting/netsuite/export/NetSuiteExportExpensesPage.tsx

```diff
@@ -0,0 +1,173 @@
+        },
+        {
+            description: translate('workspace.netsuite.nonReimbursableJournalPostingAccount'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT_EXPENSES_PAYABLE_ACCOUNT_SELECT.getRoute(policyID, params.expenseType)),
+            brickRoadIndicator: config?.errorFields?.payableAcct ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: selectedPayableAccount ? selectedPayableAccount.name : undefined,
+            pendingAction: config?.pendingFields?.payableAcct,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.PAYABLE_ACCT),
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.PAYABLE_ACCT),
+            shouldHide: isReimbursable || exportDestination !== CONST.NETSUITE_EXPORT_DESTINATION.JOURNAL_ENTRY,
+        },
+        {
+            description: translate('workspace.netsuite.reimbursableJournalPostingAccount'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT_EXPENSES_PAYABLE_ACCOUNT_SELECT.getRoute(policyID, params.expenseType)),
+            brickRoadIndicator: config?.errorFields?.reimbursablePayableAccount ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: selectedReimbursablePayableAccount ? selectedReimbursablePayableAccount.name : undefined,
+            pendingAction: config?.pendingFields?.reimbursablePayableAccount,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.REIMBURSABLE_PAYABLE_ACCOUNT),
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.REIMBURSABLE_PAYABLE_ACCOUNT),
+            shouldHide: !isReimbursable || exportDestination !== CONST.NETSUITE_EXPORT_DESTINATION.JOURNAL_ENTRY,
+        },
+        {
+            description: translate('workspace.netsuite.journalPostingPreference.label'),
+            onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_NETSUITE_EXPORT_EXPENSES_JOURNAL_POSTING_PREFERENCE_SELECT.getRoute(policyID, params.expenseType)),
+            brickRoadIndicator: config?.errorFields?.journalPostingPreference ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
+            title: config?.journalPostingPreference ? translate(`workspace.netsuite.journalPostingPreference.values.${config.journalPostingPreference}`) : undefined,
+            pendingAction: config?.pendingFields?.journalPostingPreference,
+            errors: ErrorUtils.getLatestErrorField(config, CONST.NETSUITE_CONFIG.JOURNAL_POSTING_PREFERENCE),
+            onCloseError: () => Policy.clearNetSuiteErrorField(policyID, CONST.NETSUITE_CONFIG.JOURNAL_POSTING_PREFERENCE),
+            shouldHide: exportDestination !== CONST.NETSUITE_EXPORT_DESTINATION.JOURNAL_ENTRY,
+        },
+    ];
+
+    return (
+        <AccessOrNotFoundWrapper
```

Comment:
Any specific reason we're not using `ConnectionLayout` here?

Reply:
I did not understand. Can you elaborate?

Reply:
Are you saying the backButtonPress? I don't see fallbackRoute. Instead we should add the `onBackButtonPress` optional prop instead of using AccessNotFoundWrapper, etc.